packages/nodejs/src/environment/NodeJsEnvironment.ts

```diff
@@ -117,13 +122,33 @@ function loadVariables(env: Environment) {
     };
 }
 
+function configureCrypto(env: Environment) {
+    if (!config.installCrypto || !(env.vars.boolean("nodejs.crypto") ?? true)) {
```

Comment:
please add the three new vars to the docs above in class header. thanks. Or are they not supposed to be used by devs? Maybe even then

Reply:
Oh, I thought I did.  I added nodejs.crypto and nodejs.network, what's the third one?